src/modules/payload_deliverer/payload_deliverer.cpp

```diff
@@ -168,6 +194,18 @@ void PayloadDeliverer::handle_vehicle_command(const hrt_abstime &now,  const veh
 			_gripper.release();
 			break;
 		}
+
+		// Send IN_PROGRESS vehicle_command_ack message to alert that DO_GRIPPER command is getting processed
+		// Since QGC/AMC doesn't impose timeout for IN_PROGRESS vehicle command acks, it is suffice to send it only once.
```

Comment:
Please do not reference specific implementations, just adhere to the MAVLink spec: https://mavlink.io/en/services/command.html#long_running_commands